src/franzininho-element.ts

```diff
@@ -0,0 +1,455 @@
+import { customElement, LitElement, property, html, svg, query, css } from 'lit-element';
+import { pinsFemalePattern } from './patterns/pins-female';
+import { analog, ElementPin, i2c, spi } from './pin';
+import { SPACE_KEYS } from './utils/keys';
+
+@customElement('wokwi-franzininho')
+export class FranzininhoElement extends LitElement {
+  @property() led1 = false;
+  @property() ledPower = false;
+  @property() resetPressed = false;
+  @query('#reset-button') resetButton!: SVGCircleElement;
+
+  readonly pinInfo: ElementPin[] = [
+    { name: 'GND.1', x: 218.5, y: 23.3, signals: [{ type: 'power', signal: 'GND' }] },
+    { name: '5V.1', x: 218.5, y: 32.9, signals: [{ type: 'power', signal: 'VCC', voltage: 5 }] },
+    { name: 'PB5', x: 218.5, y: 42.5, signals: [analog(0)] },
+    { name: 'PB4', x: 218.5, y: 52.2, signals: [analog(2), { type: 'pwm' }] },
+    { name: 'PB3', x: 218.5, y: 61.7, signals: [analog(3)] },
+    { name: 'PB2', x: 218.5, y: 71.2, signals: [spi('SCK'), i2c('SCL'), analog(1)] },
+    { name: 'PB1', x: 218.5, y: 80.9, signals: [spi('MISO'), { type: 'pwm' }] },
+    { name: 'PB0', x: 218.5, y: 90.5, signals: [spi('MOSI'), i2c('SDA'), { type: 'pwm' }] },
+    { name: 'VIN', x: 132.7, y: 8.1, signals: [{ type: 'power', signal: 'VCC' }] },
+    { name: 'GND.2', x: 142.9, y: 8.1, signals: [{ type: 'power', signal: 'GND' }] },
+    { name: '5V.1', x: 153, y: 8.1, signals: [{ type: 'power', signal: 'VCC', voltage: 5 }] },
```

Comment:
Duplicate pin name. Also, I think it may make more sense to call it "VCC.2" (and the other one VCC.1) as this is the text written next to it on the PCB